storage/innobase/handler/ha_innodb.cc

```diff
@@ -17602,11 +17502,16 @@ static int innodb_ft_aux_table_validate(THD *thd, st_mysql_sys_var*,
 	int len = sizeof buf;
 
 	if (const char* table_name = value->val_str(value, buf, &len)) {
+		/* Because we are not acquiring MDL on the table name,
+		we must contiguously hold dict_sys.latch while we are
+		examining the table, to protect us against concurrent DDL. */
+		dict_sys.lock();
```

Comment:
This needs to be `dict_sys.lock(SRW_LOCK_CALL)`, or otherwise this will only work with `cmake -DPLUGIN_PERFSCHEMA=NO`, which I was using locally.